src/test/java/seedu/guest/model/guest/RoomTest.java

```diff
@@ -25,20 +25,24 @@ public void isValidRoom() {
         assertThrows(NullPointerException.class, () -> Room.isValidRoom(null));
 
         // invalid room
-        assertFalse(Room.isValidRoom("")); // empty string
-        assertFalse(Room.isValidRoom(" ")); // spaces only
+        assertFalse(Room.isValidRoom("")); // empty string, near boundary values
+        assertFalse(Room.isValidRoom(" ")); // spaces only, near boundary values
         assertFalse(Room.isValidRoom("-50-23")); // not allowed to start with a hyphen
         assertFalse(Room.isValidRoom("50 - 23")); // not allowed to have spacings in between
         assertFalse(Room.isValidRoom("50--23")); // not allowed to have more than 1 hyphen in between
         assertFalse(Room.isValidRoom("$50")); // '$' is not allowed
         assertFalse(Room.isValidRoom("^&*")); // special symbols only are not allowed
         assertFalse(Room.isValidRoom("aloha-5()")); // '(' or ')' is not allowed
```

Comment:
Not sure if all 3 test cases are needed to test symbols. Maybe just have 1 test case like "5(1)A"? What do you think?

Reply:
Sounds good!